.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: 💬 Question
+about: If you need help understanding the HumanConnection.
```

Comment:
```suggestion
about: If you need help to understand Human Connection.
```

Reply:
Actually I think grammatically it's better `need help understanding`, but I would also suggest to lose `the` 
Suggested: `If you need help understanding Human Connection`

Reply:
will adopt!